### ISSUE TITLE
theme-vertex is broken

### ISSUE THREAD

**Issue by dermetfan** [NixOS/nixpkgs#29026]
### Steps to reproduce

```
❯ nix-build -A theme-vertex '<nixpkgs>'
these derivations will be built:
  /nix/store/qwfhyry6hdz9gsml8zxd0vaqf5h82r8c-theme-vertex-20170128.drv
building path(s) ‘/nix/store/r1a80n8mkfid5qsq25smgg74npsmwhwn-theme-vertex-20170128’
unpacking sources
unpacking source archive /nix/store/b33yzs4zpvxnw2wmymcgzabwslh8sgzm-vertex-theme-20170128-src
source root is vertex-theme-20170128-src
patching sources
autoreconfPhase
autoreconf: Entering directory `.'
autoreconf: configure.ac: not using Gettext
autoreconf: running: aclocal --force -I m4
autoreconf: configure.ac: tracing
autoreconf: configure.ac: creating directory build-aux
autoreconf: configure.ac: not using Libtool
autoreconf: running: /nix/store/zly5qfsq5fy6axznsza64q83jdsbq711-autoconf-2.69/bin/autoconf --force
autoreconf: configure.ac: not using Autoheader
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:12: installing 'build-aux/install-sh'
configure.ac:12: installing 'build-aux/missing'
autoreconf: Leaving directory `.'
configuring
configure flags: --prefix=/nix/store/r1a80n8mkfid5qsq25smgg74npsmwhwn-theme-vertex-20170128 --disable-unity --with-gnome=3.24
checking for a BSD-compatible install... /nix/store/bx5zp07kbklb1ichysj5p1kx1qa846lp-coreutils-8.27/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /nix/store/bx5zp07kbklb1ichysj5p1kx1qa846lp-coreutils-8.27/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for pkg-config... /nix/store/278drd32xkvm1w29hj2z2vh28hnfivp1-pkg-config-0.29.2/bin/pkg-config
checking pkg-config is at least version 0.9.0... yes
configure: error: invalid GNOME version: 3.24
builder for ‘/nix/store/qwfhyry6hdz9gsml8zxd0vaqf5h82r8c-theme-vertex-20170128.drv’ failed with exit code 1
error: build of ‘/nix/store/qwfhyry6hdz9gsml8zxd0vaqf5h82r8c-theme-vertex-20170128.drv’ failed
```

## Technical details

* System: `18.03pre114421.799435b7ca (Impala)`
* Nix version: `nix-env (Nix) 1.11.14`
* Nixpkgs version: `18.03pre114421.799435b7ca`
* Sandboxing enabled: no

**Comment by rnhmjoj** [comment]
It appears it was fixed by 7ef6d3ced9b865a7180d58ce78ed62aad93189a0. Can we close?

**Comment by dermetfan** [comment]
Works for me. :+1:
